sps20/content/contents.lr

```diff
@@ -24,6 +24,18 @@ The conference will be held in English and is suited for all skill levels, from
 
 The summit is held on the **OST campus** in the beautiful town of **Rapperswil**. Find more information on the [venue](/venue) page.
 
+### Sponsors
+
+#### Gold
+
+#### Silver
+
+<a href="https://deplo.io"><img src="/sponsors/deploio.svg" style="height: 80px; width: 80px"></a>
+
+#### Bronze
+
+Do you want to support Swiss Python Summit? Find out more about [sponsoring opportunities](./sponsoring#sponsoring-2024).

```

Comment:
Maybe put this sentences after the ### Sponsors and not under #### Bronze ? But I if you have it there for a reason, let it as it is and merge it

Reply:
The reasoning was to have it in the end after all sponsors, but I see that it doesn't make much sense this way. and looks a bit confusing.